lib/optimizely/decision_service.rb

```diff
@@ -123,11 +128,7 @@ def get_variation_for_feature(feature_flag, user_id, attributes = nil)
           Logger::INFO,
           "User '#{user_id}' is bucketed into a rollout for feature flag '#{feature_flag_key}'."
         )
-        # return decision with nil experiment so we don't track impressions for it
-        return {
-          'experiment' => nil,
-          'variation' => variation
-        }
+        return variation
```

Comment:
* Rename variation to decision. This will cause confusion. The get_Variation_for_feature_rollout now returns decision
* Also change the method return definition, the method now returns a Decision struct and not a hash.